src/main/java/lottery/strategy/RandomTicketIssueStrategy.java

```diff
@@ -0,0 +1,39 @@
+package lottery.strategy;
+
+import lottery.domain.LotteryNumber;
+import lottery.domain.LotteryTicket;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static lottery.Constant.*;
+
+public class RandomTicketIssueStrategy implements TicketIssueStrategy {
+    private static final List<LotteryNumber> allLotteryNumbers;
+
+    static {
+        allLotteryNumbers = new ArrayList<>();
+        for (int i = LOTTERY_MIN_NUMBER; i < LOTTERY_MAX_NUMBER; i++) {
+            allLotteryNumbers.add(new LotteryNumber(i));
+        }
+    }
+
+    private static List<LotteryNumber> selectLotteryNumbers() {
+        List<LotteryNumber> lotteryNumbers = new ArrayList<>();
+        for (int i = 0; i < LOTTERY_TICKET_SIZE; i++) {
+            lotteryNumbers.add(allLotteryNumbers.get(i));
+        }
+        return lotteryNumbers;
+    }
+
+    private static void shuffle() {
+        Collections.shuffle(allLotteryNumbers);
+    }
+
+    @Override
+    public LotteryTicket issue() {
+        shuffle();
```

Comment:
```suggestion
        Collections.shuffle(allLotteryNumbers); 
```

이렇게 작성하는것과 유의미한 차이가 없다면 굳이 메서드로 분리 안하셔도 됩니다 ㅎㅎ